osschallenge/migrations/0001_initial.py

```diff
@@ -27,13 +27,6 @@ class Migration(migrations.Migration):
                 ('author', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
             ],
         ),
-        migrations.CreateModel(
-            name='Groups',
-            fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('group', models.CharField(max_length=50)),
-            ],
-        ),
```

Comment:
Is it intentional to update migrations which might already be applied on a database?

You should instead let django create a migrations and do not update existing migrations. What you can do now is to revert the change in this file and run `./manage.py makemigrations` again.